source/includes/_bundle.md

```diff
@@ -530,15 +533,18 @@ bundle.rename_external_key(user,
 ```
 
 ```python
-bundleApi = killbill.api.BundleApi()
-body = Bundle(bundle_id=bundle_id, 
+created_by = 'user'
+
+bundleApi = killbill.BundleApi()
+
+bundleID = 'd65d22aa-483e-45fb-9335-cb6636514479'
+
+body = Bundle(bundle_id=bundleID,
               external_key='new_external_key')
 
-bundleApi.rename_external_key(bundle_id,
+bundleApi.rename_external_key(bundleID,
                               body,
-                              created_by,
-                              api_key,
-                              api_secret)
+                              created_by)
```

Comment:
Could you add the `created_by='demo', reason='reason', comment='comment'` parameters like done in[ other API endpoints](https://killbill.github.io/slate/account.html#create-an-account) for the sake of consistency?

Reply:
This would also need to be done for other endpoints as required.